src/rtScript.h

```diff
@@ -98,9 +106,21 @@ class rtScript
 
   void* getParameter(rtString param);
 
+#ifdef ENABLE_DEBUG_MODE
+  rtError enableDebugger(bool);
+#endif
+
 private:
+#ifdef ENABLE_DEBUG_MODE
```

Comment:
Rather than defining methods or not defining methods depending on build defines, let's instead always define these methods, and let's have the methods themselves log a message if the ability is disabled and take no action.  Please follow the same pattern for all method declarations and definitions.